.github/workflows/master.yml

```diff
@@ -40,24 +40,33 @@ jobs:
     steps:
       - name: Checkout
         uses: actions/checkout@v2
+      - uses: unionai/flytectl-setup-action@v0.0.1
+        name: Setup flytectl
       - id: load-docker-cache
         name: Load Docker Cache
         uses: actions/cache@v1
         with:
           path: /tmp/tmp/docker-images
           key: /tmp/docker-images-${{ github.event.after }}
+      - name: Create Sandbox Cluster
+        run: |
+          cp /tmp/tmp/docker-images/snapshot.tar snapshot.tar
+          flytectl config init
+          flytectl sandbox start --source=$(pwd)
       - name: Prime docker cache
-        run: docker load -i /tmp/tmp/docker-images/snapshot.tar || true
-      - uses: engineerd/setup-kind@v0.5.0
-        with:
-          version: "v0.11.1"
+        run: |
+          flytectl sandbox exec -- docker load -i /root/snapshot.tar
+      - name: Upgrade Helm charts
+        run:  flytectl sandbox exec -- helm upgrade flyte -n flyte --kubeconfig=/etc/rancher/k3s/k3s.yaml /flyteorg/share/flyte -f /flyteorg/share/flyte/values-sandbox.yaml --set flyteadmin.image.repository=flyteorg/flyteadmin,flyteadmin.image.tag=latest
```

Comment:
This should be able to run without `flytectl sandbox exec` right? no need to change it... just validating my understanding